automation/destructive.sh

```diff
@@ -18,7 +18,7 @@
 #
 
 set -ex
-export TARGET=k8s-1.26-centos9
+export TARGET=k8s-1.28
```

Comment:
wdyt about running here
`find cluster-provision/k8s/* -maxdepth 0 -type d -printf '%f\n'`
and taking the last line ?

and on the 1.27 ones take one before last line
and then it aint need to be changed anymore

btw if you prefer that the code will show the selected version instead an expression
then maybe this can be a helper script that replaces all the needed places in this commit, based the the above expression

Reply:
I prefer to leave this manual, since I don't think we ever defined the version calculation formally, and I don't want to rely on the folder structure changing in the future

Reply:
folder structure didnt change for years, and i dont think will, but up to you